### PR TITLE
Make meshery logo consistent.

### LINKED ISSUE #156: [Readme] Consistently present Meshery logo
#### Current State

1> The logo in the readme is not consistent with the one in meshery/meshery repo make it consistent :) 

2> "Join Layer5 Slack!" text in the readme doesn't display properly in dark mode. Fix it.
#### Desired State

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📝 [Meshery Adapters Spreadsheet](https://docs.google.com/spreadsheets/d/1rGGpSXC68iDJzNRU-qZIRQsZUwkt1qLFg7JCs4pfvbU/edit?usp=sharing)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by NPDebs**
**Description**
This PR fixes #156

Screenshot_20220928-121402_Chrome

**Notes for Reviewers**
The meshery logo now looks legible in both light and dark mode. The same goes for the Slack logo.
Thank you.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by legendarykamal — APPROVED** [review]
yo looks sleek ! thank you @NPDebs :+1:

**Comment by NPDebs** [comment]
Thank you! 🙌🏼💃🏽

**Review by Nikhil-Ladha — COMMENTED** [review]
Please copy the images to this repo and use the relative path to the images in the README

**Comment by NPDebs** [comment]
Sure. I'll get right on it! 🙌🏼

**Comment by NPDebs** [comment]
@Nikhil-Ladha & @legendarykamal ...everything looks good on my end. Could you kindly review again? Thank you.

File: README.md [hunk 1]
```diff
@@ -35,9 +43,17 @@ NGINX Service Mesh (NSM), a fully integrated lightweight service mesh that lever
 <h2><a name="contributing"></a><a name="community"></a> <a href="http://slack.layer5.io">Community</a> and <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">Contributing</a></h2>
 Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>! Contributors are expected to adhere to the <a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Code of Conduct</a>.
 
-<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="img/readme/slack-128.png" style="margin-left:10px;padding-top:5px;" width="110px" align="right" /></a>
+<a href="https://meshery.io/community">
```
Comment (legendarykamal) [inline README.md:46]: Oops Wrong redirection, this should redirect to joining slack.

File: README.md [hunk 2]
```diff
@@ -35,9 +43,17 @@ NGINX Service Mesh (NSM), a fully integrated lightweight service mesh that lever
...
+
+<picture align="right">
+  <source media="(prefers-color-scheme: dark)" srcset="./img/readme/slack-dark-128.png"  width="110px" align="right" style="margin-left:10px;margin-top:10px;">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/readme/slack-128.png" width="110px" align="right" style="margin-left:10px;padding-top:5px;">
+  <img alt="Shows an illustrated light mode meshery logo in light color mode and a dark mode meshery logo dark color mode." src="/img/readme/slack-128.png" width="110px" align="right" style="margin-left:10px;padding-top:13px;">
+</picture>
+</a>
 
-<a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src="img/readme/community.svg" style="margin-right:8px;padding-top:5px;" width="140px" align="left" /></a>
+<a href="http://slack.layer5.io">
```
Comment (legendarykamal) [inline README.md:55]: and this should actually redirect to https://meshery.io/community

File: README.md [hunk 3]
```diff
@@ -35,9 +43,17 @@ NGINX Service Mesh (NSM), a fully integrated lightweight service mesh that lever
...
 Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>! Contributors are expected to adhere to the <a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Code of Conduct</a>.
 
-<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="img/readme/slack-128.png" style="margin-left:10px;padding-top:5px;" width="110px" align="right" /></a>
+<a href="https://meshery.io/community">
+
+<picture align="right">
+  <source media="(prefers-color-scheme: dark)" srcset="./img/readme/slack-dark-128.png"  width="110px" align="right" style="margin-left:10px;margin-top:10px;">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/readme/slack-128.png" width="110px" align="right" style="margin-left:10px;padding-top:5px;">
```
Comment (legendarykamal) [inline README.md:50]: This also comes from meshery repo, please add the logo in this repo only.

**Review by legendarykamal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NPDebs) [reply README.md:46]: Shoot. 🙈 Thanks for spotting that out! 🙌🏼

File: README.md (hunk 2 above)
Comment (NPDebs) [reply README.md]: Done. Thank you. 🤝🏻

File: README.md (hunk 3 above)
Comment (NPDebs) [reply README.md]: All done. 🙂

**Comment by NPDebs** [comment]
@legendarykamal I've made the necessary changes. Thanks for lending your keen eyes. 👀 👍🏻

File: README.md [hunk 4]
```diff
@@ -1,12 +1,20 @@
-<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><img align="center" style="margin-bottom:20px;" src="https://raw.githubusercontent.com/layer5io/meshery-nginx-sm/master/img/readme/meshery-logo-tag-light-text-side.svg"  width="70%" /></a><br /><br /></p>
+<p style="text-align:center;" align="center">
+<a href="https://layer5.io/meshery">
+<picture align="center">
+<source media="(prefers-color-scheme: dark)" srcset="./img/readme/meshery-logo-dark-text-side.svg" width="70%" align="center" style="margin-bottom:20px;">
+<source media="(prefers-color-scheme: light)" srcset="./img/readme/meshery-logo-tag-light-text-side.svg" width="70%" align="center" style="margin-bottom:20px;">
```
Comment (legendarykamal) [inline README.md:5]: @leecalcote and we have agreed to get rid of that tagline present in the light mode. Please replace it , see meshery/meshery for ref on which one to use.

**Review by legendarykamal — CHANGES_REQUESTED** [review]
All looks fine just one more change needed.

File: README.md (hunk 4 above)
Comment (NPDebs) [reply README.md:5]: True, true.
It seems off that one mode has the tagline when another doesn't. 🤔

I'll fix that right away! 🤝🏻

**Comment by NPDebs** [comment]
All done @legendarykamal || @Nikhil-Ladha 👍🏻

**Comment by NPDebs** [comment]
👀 Subtle reminder that this still exists. 🌝|| @legendarykamal @Nikhil-Ladha

**Review by legendarykamal — APPROVED** [review]
LGTM 🚀

**Review by Nikhil-Ladha — APPROVED** [review]
Lgtm

**Comment by legendarykamal** [comment]
yeaaa @NPDebs 👏

**Pull request merged**